docs/source/usingmame/mamemenus.rst

```diff
@@ -1,7 +1,9 @@
 MAME Menus
 ==========
 
-If you started MAME without any command line parameters, you'll be shown the game selection menu immediately. While the keys listed above will let you navigate the menus, you can also use a mouse.
+If you started MAME without any command line parameters, you'll be shown the
+driver selection menu immediately. While the keys listed above will let you
```

Comment:
It’s called the “system selection menu”.

Reply:
Ah, another legacy instance of 'game'. Fixed.